CMakeLists.txt

```diff
@@ -85,28 +89,12 @@ if (NOT NO_SDL2)
 
 	if (SDL2_FOUND)
 		message(STATUS "SDL2 found! Enabling UI components.")
-		include_directories(${SDL2_INCLUDE_DIR})
-
-		if (MSVC)
-  			target_link_libraries(${PROJECT_NAME} ${SDL2_LIBRARY})
-		else()
-  			target_link_libraries(${PROJECT_NAME} -lpthread -lm ${SDL2_LIBRARY})
-		endif()
-		add_definitions(-DCRAY_SDL_ENABLED)
```

Comment:
This definition is needed for C-ray to use SDL. Unless CMake has a more standard way of telling the preprocessor that a dependency is indeed available?

Reply:
Good catch, I will add that back.